examples/playground/src/app.js

```diff
@@ -128,6 +130,29 @@ export class App extends Component {
 
   render() {
     const {jsonProps, initialViewState} = this.state;
+
+    let deckMap;
+    if (jsonProps.google === true) {
```

Comment:
I think there should be a little discussion about what the prop name should be called. Especially because it's possible more basemap providers could be added in the future, e.g. ArcGIS/Harp.gl.

So what about a top-level property `basemapProvider` that defaults to `Mapbox` if not provided, and when
```
"basemapProvider": "Google"
```
you switch to the Google basemap

Reply:
Sympathetic to this--we should do an API review and decide what this looks like

Reply:
Maybe you could prep a playground API review task with bullets from comments here?